assets/prefabs/MagicFool.prefab

```diff
@@ -1,45 +1,297 @@
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": [
+              {
+                "class": "Dialogs:CloseDialogAction"
+              },
+              {
+                "class": "Dialogs:NewDialogAction",
+                "target": "dome"
+              }
+            ]
+          },
+          {
+            "text": "How can I take the other team's flag?",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": [
+              {
+                "class": "Dialogs:CloseDialogAction"
+              },
+              {
+                "class": "Dialogs:NewDialogAction",
+                "target": "flag"
+              }
+            ]
+          },
+          {
+            "text": "How to win and score points?",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": [
+              {
+                "class": "Dialogs:CloseDialogAction"
+              },
+              {
+                "class": "Dialogs:NewDialogAction",
+                "target": "score"
+              }
+            ]
+          },
+          {
+            "text": "Back",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": [
+              {
+                "class": "Dialogs:CloseDialogAction"
+              },
+              {
+                "class": "Dialogs:NewDialogAction",
+                "target": "main"
+              }
+            ]
+          },
+          {
+            "text": "Close",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": {
+              "class": "Dialogs:CloseDialogAction"
+            }
+          }
+        ]
+      },
+      {
+        "id": "team",
+        "title": "How to join a team?",
+        "paragraphText": [
+          "When you interact with any of the dice teleporters on this platform, you'll join the related red or black team and spawn in the gameplay arena on your team's home base."
+        ],
+        "responses": [
+          {
+            "text": "Back",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": [
+              {
+                "class": "Dialogs:CloseDialogAction"
+              },
+              {
+                "class": "Dialogs:NewDialogAction",
+                "target": "instructions"
+              }
+            ]
+          },
+          {
+            "text": "Close",
+            "responseImage": "LightAndShadowResources:answerArrow",
+            "action": {
+              "class": "Dialogs:CloseDialogAction"
+            }
+          }
+        ]
+      },
+      {
+        "id": "whatisdome",
+        "title": "What is magic dome?",
+        "paragraphText": [
+          "When you will get spawned in the gameplay arena, you will notice a magic dome around you that you cannot escape, same goes for the other team on the other side of the gameplay arena."
```

Comment:
The text doesn't match the title as it doesn't really explain what the magic dome is but only says that it will be there.

Reply:
so should I remove this button or add something else in its text?